packages/opc-base/src/components/opc-footer/opc-footer.ts

```diff
@@ -0,0 +1,27 @@
+import { LitElement, html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+import style from "./opc-footer.styles";
+
+@customElement("opc-footer")
```

Comment:
`opc-footer` can conflict with the other `opc-footer` component from the `op-components` library.
Can you rename this to something more specific to it's use case? (eg. `opc-drawer-footer` or something similar)

Reply:
Done